lib/confluence.js

```diff
@@ -336,4 +336,77 @@ Confluence.prototype.updateAttachmentData = function(space, id, attachmentId, fi
         });
 };
 
+/**
+ *  This allows you to search by query
+ *
+ * @param {string} query
+ * @param {Function} callback
+ */
+Confluence.prototype.search = function(query, callback){
```

Comment:
The search function looks like a great addition. Thanks!

Reply:
@johnpduane 
Hi John, I see that the search function has been approved but I dont see it in the master branch, is it possible to add it in?